docs/CONTRIBUTING.md

```diff
@@ -5,60 +5,129 @@ route: /CONTRIBUTING.md
+more guidance on the kinds of things you might identify in an issue.
 
-- **_Bugs_**: Any problems that you've found in either the `Atlantis` build
-  code, or in its components
-- **_Feature Request_**: Requests for any features you'd like to see in the
-  `Atlantis` design system. New components requests, Api changes to components,
-  Improvements to the docs, etc...
-- **_Enhancement_**: Requests for any changes to the build infrastructure for
-  `Atlantis`, such as changes to our linting rules, proposing a mono-repo
-  structure, requesting changes to our testing tools, etc...
+To submit an issue, go to our
+[Github new issue page](https://github.com/GetJobber/atlantis/issues/new) and
+fill out the details.
 
-## Write a proposal
+### Write a proposal
 
-**_What should be in a proposal?_**
+#### I know what I want to add, can I just put up a PR?
 
-Please use the
-[proposal template](https://github.com/GetJobber/atlantis/blob/master/docs/proposals/TEMPLATE.md)
-as a guide for writing proposals.
+The proposal is an opportunity to step outside of your immediate needs and
+consider things like:
 
-**_What is the process for submitting a proposal?_**
+- What _is_ this component... what action does it enable the user to take?
+- What else could this be used for?
+- How have other products/design systems solved this type of component?
+- How can we ensure this component is accessible?
+- What properties will be needed for the next person to use this component?
 
-1. Create a branch named `<component_name>-proposal`
-2. Copy the proposal template and place it
-   under `docs/proposals/<component_name>/README.md`
-3. Fill out the proposal template
-4. Submit a PR for the propsoal
-5. Gather feedback and iterate on proposal
-6. Once approved a maintainer will land the proposal in `master` and they'll
-   create an `Issue` ticket to implement the proposal.
+It's also an opportunity for the Atlantis team to consider how this contribution
+might impact other elements of the system, and offer guidance on best practices
+and established patterns.
 
-## Implement a Proposal
+#### What should be in a proposal?
 
-All components should start with a proposal. One of our
-[Guiding Principles](#guiding-principles) is to start all components with
-documentation. If you would like to implement a component, that does not have a
-proposal, please see [Write a proposal](#write-a-proposal).
+A component's proposal should take the form of a component's documentation. In
+this way, we're describing the intended behaviour and functionality of the
+component, as well as how it fits within the Atlantis system, to determine
+requirements before we build.
 
-**_What's the process?_**
+The documentation requirements are captured in our
+[documention template](https://github.com/GetJobber/atlantis/blob/master/packages/generators/templates/component/%7B%7Bname%7D%7D.%7B%7Bmdx%7D%7D)
+and cover the following topics:
```

Comment:
mayeb we should mention here that this is generated as part of `npm run generate` so that people are not just copy/pasting this mdx file.

Reply:
Changed this section to promote use of the generator to _get_ the template, rather than a link to look at the generator template itself 👍